src/main/java/com/github/pedrovgs/androidwifiadb/adb/ADB.java

```diff
@@ -28,6 +29,7 @@
   private final CommandLine commandLine;
   private final ADBParser adbParser;
   private Project project;
+  private boolean isTCPEnabled = false;
```

Comment:
We should not keep if the tpc connection has been already initialized. The plugin user can disable this connection from the command line and the plugin will not be able to know if the tpc mode is connected or not. Is there any reason to use this attribute to check the tcp connection status?

In addition, two instances of this ADB class will not share the state. If we want to check if the tcp mode is enabled or not you should execute a command, not keep the state here.